.gitignore

```diff
@@ -1,14 +1 @@
-*.iml
-.gradle
-/local.properties
-/.idea/caches
-/.idea/libraries
-/.idea/modules.xml
-/.idea/workspace.xml
-/.idea/navEditor.xml
```

Comment:
why are these removed?

Reply:
I think these were removed by accident when I added secrets.xml to the file- will fix this!